src/libs/interfaces/SkillerInterface.xml

```diff
@@ -26,7 +26,7 @@
     <field type="string" length="128" name="error">
       String describing the error. Can be set by a skill when it fails.
     </field>
-    <field type="uint32" name="exclusive_controller">
+    <field type="string" length="37" name="exclusive_controller">
```

Comment:
maybe add to the description that the 37 is a null-terminated uuid so people do not stumble across the number falling from the sky.